lib/changes_are_logged.rb

```diff
@@ -43,15 +53,21 @@ def save_change_log
     def attribute_change(attr)
       [changed_attributes[attr], __send__(:read_attribute, attr)] if attribute_changed?(attr)
     end
+  end
+
+  module ClassMethods
+    attr_reader :log_changes_options
 
-    def automatically_log_changes
-      @log_changes = true
+    def automatically_log_changes(options = {})
+      after_initialize -> { @log_changes = true }
+      @log_changes_options = options
```

Comment:
IMO yes, since the `after_initialize` block *will* be inherited, but the options won't be.